src/locales/default.json

```diff
@@ -1,3 +1,70 @@
 {
-  "createSpace": "Create space"
+  "searchPlaceholder": "Search",
+  "spaces": "space(s)",
```

Comment:
We should always store the complete sentence here, for such text you can add variable like this: 
```suggestion
  "spaces": "{0} space(s)",
```

Reply:
Oh I see